app/views/main.html

```diff
@@ -11,7 +11,7 @@
             <form class="form">
               <div class="form-group">
                 <!--<h3 class="text-center"><label for="speciesName">Explore Interactions of...</label></h3>-->
-                <h3 class="text-center">{{query.interaction | uppercase }} Network of</h3>
+                <h3 class="text-center">{{query.interaction | uppercase }} Exploration of</h3>
```

Comment:
@psmacmur I made a small text change on the explore page, because the word "Network" was a little confusing here.